pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -27,11 +27,13 @@ import (
 
 const (
 	// defaultHelperImage is default image of helper
-	defaultHelperImage     = "busybox:1.26.2"
-	defaultTimeZone        = "UTC"
-	defaultExposeStatus    = true
-	defaultSeparateSlowLog = true
-	defaultEnablePVReclaim = false
+	defaultHelperImage        = "busybox:1.26.2"
+	defaultTimeZone           = "UTC"
+	defaultExposeStatus       = true
+	defaultSeparateSlowLog    = true
+	defaultSeparateRocksDBLog = true
+	defaultSeparateRaftLog    = true
```

Comment:
This will cause the existing TiKV cluster to be rolling updated, we cannot set the default value to `true`.